packages/post-rector/src/Collector/NodesToAddCollector.php

```diff
@@ -103,7 +103,7 @@ public function clearNodesToAddBefore(Node $node): void
 
     private function resolveNearestExpressionPosition(Node $node): string
     {
-        if ($node instanceof Expression) {
+        if ($node instanceof Expression || $node instanceof Stmt) {
```

Comment:
@afilina This was the goled spot :laughing:

Reply:
Thanks for looking into it! There's so much going on under the hood that it's intimidating :)

Reply:
Don't worry, take your time. It took me about 6 month to start to understand php-parser and AST.
3 more months to make first usable version Rector around it.

It might take to write 10-15 Rector rules to get some confidence working with it. But then, nothing is imposible :) everything is matter of time